lib/ts/supertokens.ts

```diff
@@ -439,4 +440,25 @@ export default class SuperTokens {
         }
         throw err;
     };
+
```

Comment:
write down list of core changes based on changes made here. Ideally per API bases.